app/pom.xml

```diff
@@ -70,6 +70,11 @@
       <artifactId>payload</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
The PR description says "can be used in the future", but actually it is bundled into the executable along with JUnit. I doubt we want it